kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/locator/ProcessContainerLocatorProvider.java

```diff
@@ -0,0 +1,53 @@
+package org.kie.server.services.jbpm.locator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+import org.kie.server.api.KieServerConstants;
+import org.kie.server.services.api.ContainerLocator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ProcessContainerLocatorProvider {
+    private static final Logger logger = LoggerFactory.getLogger(ProcessContainerLocatorProvider.class);
+
+    private String processInstanceLocatorName =
```

Comment:
I think this field can be removed, see my comments below

Reply:
Done